packages/core/src/components/fields/LayoutMultiSchemaField.tsx

```diff
@@ -0,0 +1,212 @@
+import { useState, useEffect } from 'react';
+import {
+  ANY_OF_KEY,
+  CONST_KEY,
+  DEFAULT_KEY,
+  EnumOptionsType,
+  ERRORS_KEY,
+  FieldProps,
+  FormContextType,
+  getDiscriminatorFieldFromSchema,
+  hashObject,
+  ID_KEY,
+  ONE_OF_KEY,
+  optionsList,
+  PROPERTIES_KEY,
+  RJSFSchema,
+  getTemplate,
+  getUiOptions,
+  getWidget,
+  SchemaUtilsType,
+  StrictRJSFSchema,
+  UiSchema,
+} from '@rjsf/utils';
+import get from 'lodash/get';
+import has from 'lodash/has';
+import isEmpty from 'lodash/isEmpty';
+import omit from 'lodash/omit';
+import set from 'lodash/set';
```

Comment:
Is there a specific reason these imports are direct imports and not package imports? There are a couple of other places in the repo that use direct imports form `lodash` too. You can global search `lodash/`

```suggestion
import { get, has, isEmpty, omit, set } from 'lodash/get';
```

Reply:
RJSF uses imports this way because not everyone has tree-shaking set up properly and it helps reduce the size of the distribution

Reply:
That makes sense, then should all import from `lodsah` be direct imports to make it consistent? I see some imports from lodash that are package imports.

Reply:
It's not important in the tests, just the code that is packaged up and published